tasks.py

```diff
@@ -375,6 +375,14 @@ def test_module(module=None, verbosity=2):
 
 
 @task
+def jshint():
+    """Run the jshint."""
+    for dirpath, dirnames, filenames in os.walk("."):
```

Comment:
It's probably hanging because you are giving it too many files to lint. We do not need to run it on `node_modules`--those are 3rd-party libs. We only need to lint our own JS, so the command should be `jshint website/static/js/`